src/styles/_catalog.scss

```diff
@@ -1,22 +1,35 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
   max-width: 944px;
+  justify-items: center;
+  row-gap: 48px;
+  column-gap: 48px;
   margin: 0 auto;
 
+  @media (max-width:488px) {
+    grid-template-columns: repeat(1, 1fr);
+  }
+
+  @media (min-width:488px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (min-width:768px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (min-width:1024px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
```

Comment:
Better use auto-fill in repet func, because fr add extra witdh for your gric cell. It also will fix your adaptive so you wouldn't need to use media queries